app/cmd/root.go

```diff
@@ -75,6 +76,10 @@ func initConfig() {
 			configLoadErrorHandle(err)
 		}
 	}
+	config = getConfig()
+	if config != nil && config.Language != "" {
```

Comment:
`config.Language != ""` is not necessary.

Reply:
Emm...  The text will be remove